### PR TITLE
Update german translation

### PR DESCRIPTION
**Pull request by fluxxcode**
Added all missing translations for German and removed all unused ones using the `update-translation.js` script.

### CODE REVIEW

**Comment by saghul** [comment]
Ping @csett86 :-)

**Comment by csett86** [comment]
Will take a look later today, thanks

File: lang/main-de.json [hunk 1]
```diff
@@ -820,8 +820,11 @@
         "videoUnmuteBlockedDescription": "Die Kamera und Bildschirmfreigabe kann aus Überlastungsschutzgründen temporär nicht eingeschaltet werden.",
         "videoUnmuteBlockedTitle": "Kamera und Bildschirmfreigabe kann nicht aktiviert werden!",
         "viewLobby": "Lobby ansehen",
+        "viewParticipants": "Teilnehmer anzeigen",
```
Comment (csett86) [inline lang/main-de.json:823]:
```suggestion
        "viewParticipants": "Personen anzeigen",
```

File: lang/main-de.json [hunk 2]
```diff
@@ -820,8 +820,11 @@
...
         "viewVisitors": "Gäste anzeigen",
         "waitingParticipants": "{{waitingParticipants}} Personen",
+        "waitingVisitors": "In der Lobby wartende Gäste: {{waitingVisitors}}",
+        "waitingVisitorsTitle": "Das Meeting wurde noch nicht gestartet!",
```
Comment (csett86) [inline lang/main-de.json:827]:
```suggestion
        "waitingVisitorsTitle": "Die Konferenz wurde noch nicht gestartet!",
```

File: lang/main-de.json [hunk 3]
```diff
@@ -1483,16 +1494,22 @@
     },
     "visitors": {
         "chatIndicator": "(Gast)",
+        "joinMeeting": {
+            "description": "Sie sind derzeit Beobachter dieser Konferenz.",
```
Comment (csett86) [inline lang/main-de.json:1498]:
```suggestion
            "description": "Sie beobachten derzeit diese Konferenz.",
```

File: lang/main-de.json [hunk 4]
```diff
@@ -1483,16 +1494,22 @@
...
+            "raiseHand": "Hand heben",
+            "title": "Dem Meeting wird beigetreten",
```
Comment (csett86) [inline lang/main-de.json:1500]:
```suggestion
            "title": "Konferenz wird beigetreten",
```

File: lang/main-de.json [hunk 5]
```diff
@@ -1483,16 +1494,22 @@
...
+            "wishToSpeak": "Wenn Sie sprechen möchten, heben Sie bitte unten Ihre Hand und warten Sie auf die Zustimmung des Moderators"
```
Comment (csett86) [inline lang/main-de.json:1501]:
```suggestion
            "wishToSpeak": "Wenn Sie sprechen möchten, heben Sie bitte unten Ihre Hand und warten Sie auf die Zustimmung der Moderation"
```

File: lang/main-de.json [hunk 6]
```diff
@@ -1483,16 +1494,22 @@
...
             "demoteDescription": "Hierhin verschoben von {{actor}}, bitte melden Sie sich um teilzunehmen",
-            "description": "Bitte melden Sie sich um teilzunehmen",
             "noMainParticipantsDescription": "Eine Person muss die Konferenz starten. Bitte versuchen Sie es gleich noch einmal.",
             "noMainParticipantsTitle": "Diese Konferenz wurde noch nicht gestartet.",
             "noVisitorLobby": "Sie können nicht teilnehmen, solange die Lobby für diese Konferenz aktiviert ist.",
             "notAllowedPromotion": "Eine Person muss Ihre Anfrage erst erlauben.",
             "title": "Sie sind Gast in der Konferenz"
-        }
+        },
+        "waitingMessage": "Sie werden dem Meeting beitreten, sobald es live ist!"
```
Comment (csett86) [inline lang/main-de.json:1512]:
```suggestion
        "waitingMessage": "Sie werden der Konferenz beitreten, sobald sie gestartet ist!"
```

**Review by csett86 — CHANGES_REQUESTED** [review]
Thanks a lot for the translation update. I have left suggestions to keep it in sync with the other parts of the German translation

**Comment by fluxxcode** [comment]
@csett86 I have applied and rebased your suggestions. Let me know if there's anything else :)

**Review by csett86 — APPROVED** [review]
Lookgs good, many thanks!

**Comment by saghul** [comment]
Thanks folks!

**Pull request merged**